### ISSUE TITLE
how to install prompt-toolkit-2.0.0?

### ISSUE THREAD

**Issue by klydream** [jonathanslenders/python-prompt-toolkit#589]
Even i clone from github, and build, install and I got the version is still 1.0.9

**Comment by jonathanslenders** [comment]
Hi @klydream,
Can you try to make sure you have the 2.0 branch checked out? ``git checkout 2.0``.
Then run ``python setup.py install``.

**Comment by dhellmann** [comment]
Is there a roadmap I can follow for the 2.0 release? It seems like at least some of the published documentation now only works with 2.0, so it's a bit confusing to us newcomers. For example, the simplest full-screen app demo is the first thing I tried and the full_screen argument isn't understood by 1.0.15.

**Comment by MechaLynx** [comment]
I used this and it worked (make sure the python version is correct and privileges are correct for your case):

```
pip install -e git+https://github.com/jonathanslenders/python-prompt-toolkit@2.0#egg=prompt_toolkit
```

**Comment by Halcyonations** [comment]
@MechaLynx so if I wanted it as a dependency I would do 
`python
install_requires= ["-e git+https://github.com/jonathanslenders/python-prompt-toolkit@2.0#egg=prompt_toolkit"]` ?

Thanks in advance

**Comment by MechaLynx** [comment]
@Halcyonations I'm not sure about that but looking at [this answer on SO](https://stackoverflow.com/a/29524177/2838033) it seems that it would?

The tarball link is on the main repo page somewhere.

**Comment by Halcyonations** [comment]
@MechaLynx Im at a Hackathon (Massey Hacks IV) and you just saved my life, Thanks a bunch!

**Comment by MechaLynx** [comment]
@Halcyonations lol np dawg :P

**Comment by Halcyonations** [comment]
@jonathanslenders  @MechaLynx Its not working, I noticed that the main branch is now defaulted to 2.0. would making a tar of the master work. When I do the downloaded tar has the name jonathanslenders-python-prompt-toolkit-1.0.14-8-g4476b3c.tar. is 1.0.14-8 a typo?

**Comment by MechaLynx** [comment]
@Halcyonations That looks like a tarball of version 1.0.14

I think this link might work `https://api.github.com/repos/jonathanslenders/python-prompt-toolkit/tarball/2.0`

**Comment by Halcyonations** [comment]
@MechaLynx Wroks! THANKS AGAIN!

**Comment by jonathanslenders** [comment]
Hi everyone,

I just released prompt_toolkit 2.0.1.
Check the changelog: https://github.com/jonathanslenders/python-prompt-toolkit/blob/master/CHANGELOG
and the docs: https://python-prompt-toolkit.readthedocs.io/  .

You can now do a pip-install.
